library/alloc/src/sync.rs

```diff
@@ -2300,6 +2300,19 @@ impl<T: ?Sized + Hash> Hash for Arc<T> {
 
 #[stable(feature = "from_for_ptrs", since = "1.6.0")]
 impl<T> From<T> for Arc<T> {
+    /// Converts a generic type `T` into an `Arc<T>`
```

Comment:
I don't think we should be using the phrase "generic type `T`" here. E.g. for things like `Vec::push()` we also don't say "Appends a generic type `T` to the back of a collection.", but just say "an element" or "a value" or possibly "a `T`".

Reply:
Same above:
https://github.com/rust-lang/rust/blob/5ea19239d9d6f49fdd76513a36386d7e83708e3f/library/alloc/src/boxed.rs#L1212

I will create a PR changing the wording soon.